### PR TITLE
added: support for ids in string.

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: src/database/migrations/20240716111210-update-user-id-to-string.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+			})
+		} catch (error) {
+			console.log(error)
+			throw error
+		}
+	},
+
+	down: async (queryInterface, Sequelize) => {
+		// Revert changes back to INTEGER
+		await queryInterface.changeColumn('availabilities', 'user_id', {
```
Comment (rakeshSgr) [inline src/database/migrations/20240716111210-update-user-id-to-string.js:55]: @nevil-mathew set allowNull: false

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -183,7 +183,7 @@ services:
   #     - elevate_net
 
   citus:
-    image: citusdata/citus:11.2.0
+    image: citusdata/citus:12.1
```
Comment (rakeshSgr) [inline docker-compose-dev.yml:186]: @nevil-mathew why this is changed?

File: src/database/migrations/20240716111210-update-user-id-to-string.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		try {
+			// Change columns to STRING
+			await queryInterface.changeColumn('availabilities', 'user_id', {
+				type: Sequelize.STRING,
```
Comment (rakeshSgr) [inline src/database/migrations/20240716111210-update-user-id-to-string.js:8]: @nevil-mathew check is there any test case we have ?  if its present, do we need to change in mock data in that case
@nevil-mathew API doc request and response check if there any change required

File: docker-compose-dev.yml (hunk 2 above)
Comment (nevil-mathew) [reply docker-compose-dev.yml:186]: To keep parity with DEV/QA server.

File: src/database/migrations/20240716111210-update-user-id-to-string.js (hunk 1 above)
Comment (nevil-mathew) [reply src/database/migrations/20240716111210-update-user-id-to-string.js:55]: updated

**Pull request merged**